stf/Cargo.toml

```diff
@@ -34,119 +34,38 @@ std = [
-branch = "master"
-optional = true
-
-[dependencies.sgx-externalities]
-default-features = false
-git = "https://github.com/scs/sgx-runtime"
-branch = "master"
-optional = true
-
-[dependencies.my-node-runtime]
-git = "https://github.com/scs/substraTEE-node"
-branch = "master"
-package = "substratee-node-runtime"
-optional = true
-
-[dependencies.substrate-api-client]
-git = "https://github.com/scs/substrate-api-client"
-branch = "master"
-optional = true
-
-[dependencies.substrate-client-keystore]
-git = "https://github.com/scs/substrate-api-client"
-branch = "master"
-optional = true
-
-[dependencies.sc-keystore]
-git = "https://github.com/paritytech/substrate.git"
-branch = "master"
-version = "3.0.0"
-optional = true
-
-[dependencies.substratee-worker-primitives]
-path = "../primitives/worker"
-default-features = false
-optional = true
-
-[dev-dependencies.sp-keyring]
-git = "https://github.com/paritytech/substrate.git"
-branch = "master"
-version = "3.0.0"
+substratee-worker-primitives = { default-features = false, path = "../primitives/worker", optional = true }
+
+# Substrate dependencies
+sp-core = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master", features = ["full_crypto"] }
+balances = { version = "4.0.0-dev", package = 'pallet-balances', default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master" }
+system = { version = "4.0.0-dev",  package = "frame-system", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master" }
+support = { version = "4.0.0-dev",  package = "frame-support", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master" }
+sp-application-crypto = { version = "4.0.0-dev", default-features = false, features = ["full_crypto"], git = "https://github.com/paritytech/substrate.git", branch = "master" }
+sp-runtime = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master" }
+sc-keystore = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master", optional = true }
+
+# scs / integritee
+sp-io = { default-features = false, features = ["disable_oom", "disable_panic_handler", "disable_allocator", "sgx"], git = "https://github.com/integritee-network/sgx-runtime", branch = "version-upgrade-f651d45", optional = true }
+sgx-runtime = { default-features = false, git = "https://github.com/integritee-network/sgx-runtime", branch = "version-upgrade-f651d45", optional = true }
+sgx-externalities = { default-features = false, git = "https://github.com/integritee-network/sgx-runtime", branch = "version-upgrade-f651d45", optional = true }
+my-node-runtime = { package = "integritee-node-runtime", git = "https://github.com/integritee-network/integritee-node", branch = "master", optional = true }
```

Comment:
Now that we no longer have the similar naming `subsratee-node-runtime` vs. `substrate-node-runtime`, I would like this to be renamed to `integritee-node-runtime`. But this can also wait.

Reply:
Lets do this with another PR - this PR is far too big, so name changes will be easily over looked